### ISSUE TITLE
Captcha Un-solvable

### ISSUE THREAD

**Issue by bobhwasatch** [MicrosoftDocs/microsoft-365-docs#649]
I cannot get my mail server delisted from your blacklist because it is impossible to solve the captcha on sender.office.com. My server is not blacklisted at Spamhaus or other lists, so it would be nice to know why I'm on the list to begin with. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 390d461b-7fce-d4ac-1c90-97b7b81dc0c2
* Version Independent ID: 9f8c7285-627f-13c6-1c7a-88fa7e14ed86
* Content: [Use the delist portal to remove yourself from the Office 365 blocked senders list](https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/use-the-delist-portal-to-remove-yourself-from-the-office-365-blocked-senders-lis#feedback)
* Content Source: [microsoft-365/security/office-365-security/use-the-delist-portal-to-remove-yourself-from-the-office-365-blocked-senders-lis.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/office-365-security/use-the-delist-portal-to-remove-yourself-from-the-office-365-blocked-senders-lis.md)
* Service: **o365-seccomp**
* GitHub Login: @MSFTTracyP
* Microsoft Alias: **tracyp**

**Comment by AnitaJH** [comment]
I also cannot get my mail server delisted from your blacklist because it is impossible to solve the captcha on sender.office.com. I tried the captcha dozens of times on several devices but keep getting the message "The characters didn't match the picture. Please try again." I know I've typed the characters correctly.

![2019-10-30_15-49-06](https://user-images.githubusercontent.com/57166589/67830650-8bdc0e00-fb2f-11e9-9b2d-f33f555d6eca.jpg)

**Comment by EricBobrow** [comment]
I tried dozens of times and was not able to solve the CAPTCHA. It is very frustrating, I have NEVER seen anything like this. It's as if it is deliberately set up to frustrate people and make them give up. I don't know why my emails are blocked, and then this makes matters worse when it promises a way to unblock the IP address but then prevents every honest effort to submit the info for review.

**Comment by AnitaJH** [comment]
Yes, it's bad enough that it doesn't work... but I have to keep refreshing the image because I can't for the life of me figure out what letters I'm looking at in most of them. You can also try the form listed at https://support.microsoft.com/en-us/supportrequestform/8ad563e3-288e-2a61-8122-3ba03d6b8d75 to try and get MS to remove any blocks. I've given it a go for what it's worth.

**Comment by Tech-Jamie** [comment]
Same issue, Captcha unreadable, audio unable to hear any of the words - called MS online services support - after 20 minutes the support rep was also unable to whitelist my IP and has promised to call me back....

**Comment by Jouwhost** [comment]
Same problem here. Code does not work, audio does not work.
Please fix this. It is bad enough you block my emails for no reason...Now i cannot even delist...

**Comment by HrvojeErceg** [comment]
Same here.Captcha unreadable/unsolvable, sound not recognizable.No one to contact.
Microsoft service is horrible as usual.

We are using both Google and Microsoft service, but Google has Customer support that is actually reachable immediately.

**Comment by simon-webb** [comment]
Same here, absolutely no way of solving the Captcha code either by picture or sound. Utterly ridiculous... Sort it Microsoft!!

**Comment by nic123** [comment]
I have created a request at Microsoft answers and someone is currently looking into this: 
https://answers.microsoft.com/en-us/msoffice/forum/msoffice_outlook-mso_other-mso_o365b/office-365-anti-spam-ip-delist-portal-unable-to/45461116-1801-4be9-884c-8cf5bf4953b6?tm=1572430374814

**Comment by AdminSoftTeco** [comment]
today I managed to enter the captcha, but the letter does not come to the my email

**Comment by bobhwasatch** [comment]
Yesterday I finally managed to enter the captcha and got the confirmation email. I sent a test email to verify that it was fixed. So problem solved in my case.

**Comment by AnitaJH** [comment]
Problem fixed for me too.

**Comment by EricBobrow** [comment]
I was able to submit the request (CAPTCHA was accepted) but no email has come through with instructions. I tried twice with two different email addresses. I'm not sure if the submission has been accepted, or if there is another step in the process.

**Comment by mypil** [comment]
@AnitaJH - Glad to know your issue is already resolved.
@EricBobrow - Please contact technical support by opening a **[service ticket](https://docs.microsoft.com/office365/admin/contact-support-for-business-products)** in your tenant so they can help resolve your issue. 

Thank you for engaging with the community here for the docs. Please feel free to open a new issue if you find any documentation error or if there is a specific area of the docs that we can improve on and we will be glad to help fix it.

**Comment by EricBobrow** [comment]
I don’t know what a tenant is. It sounds like an account admin for a company that maintains a contract with Microsoft 365. I am simply sending transactional emails to people who opt in for info or purchase an online training from my company, and have seen these emails bounce back as undeliverable due to my IP address being blocked. Please explain further what you recommend.

**Comment by mypil** [comment]
@EricBobrow - Below screenshot shows the instructions on how you can open a service ticket in your tenant. If you have no access to this then you may also use [Phone Support](https://docs.microsoft.com/office365/admin/contact-support-for-business-products?view=o365-worldwide&tabs=phone) or [In-store support](https://docs.microsoft.com/office365/admin/contact-support-for-business-products?view=o365-worldwide&tabs=instore). Hope this helps.

![image](https://user-images.githubusercontent.com/47188252/68066583-06499f80-fd75-11e9-8142-dff2c71a2539.png)
